codegen.ts

```diff
@@ -0,0 +1,19 @@
+import type { CodegenConfig } from '@graphql-codegen/cli'
+
+const config: CodegenConfig = {
+  overwrite: true,
+  schema: './src/graphql/data/schema.graphql',
+  documents: ['./src', '!**/node_modules/**', '!**/__mocks__/**', '!**/__generated__/**', '!**/thegraph/**'],
```

Comment:
can this be more specific?

Reply:
sure, but this was a direct port of what was in `relay.config.js`